File: loadlog.m
function [log] = loadlog(name)

disp('load logs');
if ~exist('name','var')
    rawdata = importdata('log.txt');
else 
    rawdata = importdata(name);
end    
cd bin
    parameters;
    
    clear ans caseName delta Delta delta99end domain flowType mesh numMethods p_col p_row time xc 
    
cd ..
data = rawdata.data;
log.saveNumber = data(:,1);
log.iter = data(:,2);
log.t = data(:,3);
log.dt = data(:,4);
log.cfl = data(:,5);
log.res = data(:,6:10);
log.U = data(:,11:5:end);
log.V = data(:,12:5:end);
log.W = data(:,13:5:end);
log.R = data(:,14:5:end);
log.E = data(:,15:5:end);


end

File: Numerical/Universal/python/loadlog.py
import numpy as np
import pandas as pd
import os

def loadlog(name=None):
    print('load logs')
    
    if name is None:
        rawdata = pd.read_csv('log.txt', delim_whitespace=True, header=None)
    else:
        rawdata = pd.read_csv(name, delim_whitespace=True, header=None)
       
    data = rawdata.values
    log = {}
    log['saveNumber'] = data[:, 0]
    log['iter'] = data[:, 1]
    log['t'] = data[:, 2]
    log['dt'] = data[:, 3]
    log['cfl'] = data[:, 4]
    log['res'] = data[:, 5:10]
    log['U'] = data[:, 10::5]
    log['V'] = data[:, 11::5]
    log['W'] = data[:, 12::5]
    log['R'] = data[:, 13::5]
    log['E'] = data[:, 14::5]

    return log